pkg/services/user/userimpl/user.go

```diff
@@ -82,25 +82,20 @@ func (s *Service) Create(ctx context.Context, cmd *user.CreateUserCommand) (*use
 		SkipOrgSetup: cmd.SkipOrgSetup,
 	}
 	orgID, err := s.orgService.GetIDForNewUser(ctx, cmdOrg)
-	cmd.OrgID = orgID
 	if err != nil {
 		return nil, err
 	}
-
 	if cmd.Email == "" {
 		cmd.Email = cmd.Login
 	}
-	usr := &user.User{
-		Login: cmd.Login,
-		Email: cmd.Email,
-	}
-	usr, err = s.store.Get(ctx, usr)
-	if err != nil && !errors.Is(err, user.ErrUserNotFound) {
-		return usr, err
+
+	err = s.store.LoginConflict(ctx, cmd.Login, cmd.Email, s.cfg.CaseInsensitiveLogin)
```

Comment:
I see that we didn't pass the `OrgID` even before but I'm curious why? I would expect to be fine to have users with the same login in different orgs, no?

Reply:
I do not know what the expected behavior is, sorry. I am not comfortable changing what's already in there if I'm not certain it's a bug but I could open a new issue to check on that?

Reply:
maybe @grafana/grafana-authnz-team is more suitable to answer that